source/functions/postProcess.m

```diff
@@ -25,6 +25,21 @@
     if iBod == 1; bodiesOutput = body1_out; end
     bodiesOutput(iBod) = eval(['body' num2str(iBod) '_out']);
     eval(['clear body' num2str(iBod) '_out'])
+end
+% Add hydrostatic and FK pressures to bodiesOutput if required.
+for iBod = 1:length(body(1,:))
+    if simu.nlHydro~=0 && body(iBod).nhBody==0 && simu.pressureDis == 1 
+        % hydrostatic pressure

```

Comment:
Note that this condition doesn't allow the pressure on the body to be output unless the nonlinear hydro option is also on. This is not what the simu.pressureDis option implies in the simulationClass

Reply:
@akeeste that's a good point. We can only provide the option to output pressure on the body if non-linear hydro is on, so we should add a warning flag letting users knows pressure will not be provided if `simu.nlHydro == 0` & simu.presssureDis ==1`.